plugins/obs-qsv11/common_utils.h

```diff
@@ -162,10 +162,12 @@ int GetFreeTaskIndex(Task *pTaskPool, mfxU16 nPoolSize);
 mfxStatus Initialize(mfxVersion ver, mfxSession *pSession,
 		     mfxFrameAllocator *pmfxAllocator, mfxHDL *deviceHandle,
 		     bool bCreateSharedHandles, bool dx9hack,
-		     enum qsv_codec codec); //vpl change
+		     enum qsv_codec codec, void **data); //vpl change
 
-// Release resources (device/display)
+// Release global shared resources (device/display)
```

Comment:
Does this actually release device/display now? I think the comment is outdated.

Reply:
This is still correct, it does the same thing it always did.